lambda/src/main/java/com/lambda/IssueUpdater.java

```diff
@@ -57,22 +65,50 @@ public Issue setActualHours(final int issueId) {
                     LocalDateTime.now());
         }
         DecimalFormat df = new DecimalFormat("0.0");
-        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60)));
+        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60.0)));
 
         if (actualHours > 999 || actualHours < 0)
             return null;
 
         return client.updateIssue(new UpdateIssueParams(issue.getId()).actualHours(actualHours));
     }
 
+    private Duration calculateWorkingHours(LocalDateTime start, LocalDateTime end) {
+        LocalDateTime current = start;
+        Duration totalWorkingDuration = Duration.ZERO;
+
+        while (current.isBefore(end)) {
+            if (isWeekday(current)) {
+                LocalDateTime endOfDay = current.withHour(19).withMinute(30); // End at 17:30 ITC as 19:30 JST
+                
+                // Add hours for current day or until end time
+                LocalDateTime actualEnd = isSameDate(end, endOfDay) ? end : endOfDay;
+                totalWorkingDuration = totalWorkingDuration.plus(Duration.between(current, actualEnd));
+            }
+
+            // Move to the next day at 09:30 JST
+            current = current.plusDays(1).withHour(9).withMinute(30);
+        }
+
+        return totalWorkingDuration;
+    }
+
+    private boolean isWeekday(LocalDateTime dateTime) {
+        return !WEEKENDS.contains(dateTime.getDayOfWeek());
+    }
+
+    private boolean isSameDate(LocalDateTime dateTime1, LocalDateTime dateTime2) {
```

Comment:
[suggestion] I prefer `isSameDay` to show we compare by day-level.

Using `Date` can sometimes imply a broader context, potentially including month or year, so `isSameDay` could help make the purpose of day-level comparison clearer to developers.

Reply:
This method to check if two dates are the same: same day, month, and year.

Reply:
ChatGPT helps me translate my intention:

> Tên phương thức isSameDate là chính xác về mặt ngữ pháp và dễ hiểu. Tuy nhiên, để rõ ràng hơn, isSameDay có thể là lựa chọn tốt hơn trong ngữ cảnh này, vì nó truyền đạt chính xác ý định kiểm tra xem hai đối tượng LocalDateTime có rơi vào cùng một “ngày” hay không.
> 
> Sử dụng Date đôi khi có thể mang hàm ý rộng hơn, bao gồm cả tháng hoặc năm, vì vậy isSameDay có thể giúp mục đích so sánh ở cấp độ ngày trở nên rõ ràng hơn đối với các lập trình viên.
> 
> Cả hai tên đều có thể chấp nhận được, nhưng isSameDay có thể được ưu tiên hơn để nhấn mạnh ý định một cách rõ ràng hơn.

Reply:
The logic will check whether two dates are the same, meaning the same day, same month, and same year.
For example:
- isSameDate("2024/01/22", "2024/02/22") return false, but isSameDate("2024/01/22", "2024/02/22") return true